### ISSUE TITLE
FAQ for webserver setups

### ISSUE THREAD

**Issue by smilerz** [TandoorRecipes/recipes#1313]
Update/Create documentation on how to setup/troubleshoot common web server and reverse proxy configurations.
Include:
- [ ] required headers
- [ ] common errors
- [ ] how to change port
- [ ] subdomain vs subfolder

Technologies:
- [ ] NGINX
- [ ] Swag
- [ ] Apache
- [ ] Traefik

**Comment by MaxJa4** [comment]
I'd add Caddy (with a low priority) too

**Comment by MaxJa4** [comment]
[Reminder]

Fields missing for nginx-config (compared to Recipes.conf that need to be mentioned in the docs):

```
proxy_set_header X-Forwarded-Proto $scheme;
proxy_redirect http://127.0.0.1:PORT https://recipes.domain.tld;
```

**Comment by MaxJa4** [comment]
After #1435 is merged, some boxes can be checked above :)

**Comment by MaxJa4** [comment]
- [ ] Update traefik tutorial/FAQs according to findings in #1616

**Comment by jgwehr** [comment]
Hey all, I've had Tandoor running on Caddy for about a month now, so I'm confident to help contribute this documentation. Started a fork here: https://github.com/jgwehr/fork-tandoorrecipes/tree/documentation/caddy

**Comment by Sheelendra-Sharma** [comment]
please assign me

**Comment by Kiran-pro2001** [comment]
Hey, Can I work on this project!

**Comment by smilerz** [comment]
> Hey, Can I work on this project!

feel free

**Comment by jgwehr** [comment]
@tehgod Check out my Caddyfile [here](https://github.com/jgwehr/homelab-docker/blob/ac04b4913292b83c345f2b7ded8fd96d19a4cdf0/services/public/staticconfig/caddy/Caddyfile#L128). This config has been working fine for a few years now.

**Comment by dharshiniramu** [comment]
Hey can I work on this issue?

**Comment by KunalTiwari-git** [comment]
Hi! I'd like to contribute to this issue. I can start by writing the NGINX setup documentation covering common errors, how to change port, and subdomain vs subfolder setup. Can this be assigned to me or is any part still available?

**Comment by smilerz** [comment]
Feel feee to submit a PR when you are ready.
